other-docs/guides/getting-help-with-altis.md

```diff
@@ -0,0 +1,47 @@
+
+We break down support into different types. Each have their own language and proccesses. But they're somewhat intuitive once you internalise the language a little. Let's go over the types.
+
+<img width="1282" alt="Screenshot-2021-01-26-at-10 14 18" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+
+In the above you can see we currently have four types of support. I’ll break down how they differ from eachother.
+
+# Incident
+
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an incident, and requires investigation.
+
+From the support side, Incidents are the things that make us scamble to respond and jump on the problem. If the site is down, this is an Incident, with a prioriy of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means neccessary.
+
+Incidents focus on immediate attention, with an emphisis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
+
+# Problem
+
+Problems are the cause of, or potential cause of an Incidents. Incidents revolve around something the unexpected. If a site is down, that’s an Incident, but the Problem is a known cause of an issue you wish to track, and coordinate help with, problems typically take longer to resolve, as an appropriate solution is thoughtout, and worked on.
+
+# Tasks 
+
+There’s an array of tasks that might you need help with. Typically this might mean provisioning of infrastucture to accomodate some need. It might be providing access to some tool or service used by the team. Or run an operation that requires a cloud engineer. Tasks should be well described.
+
+# Question (Service Requests)
+
+This type is fairly self explanatory. Have a question for us? Ask!
+
+If you need some kind of consultancy, technical question, clarification on a piece of the product, just open a support question.
+
+# Other methods of support
+
+I mentioned at the top of this page there was two main ways, and we’ve just covered the primary method. The second method is email, which is typically discouraged, but it’s useful to have thia ability when needed.
+
+We have a few email addresses linked to our support, so we can ticket issues that come in, and they’re linked to different support types.
+
+hosting@humanmade.com is the most commonly used, and this linked to an Incident type (non-urgent).
+
+support@humanmade.com is rarely used, but this is linked to the Question support type.
+
+sysops@humanmade.com is the linked to an ‘Urgent’ Incidident type, and is linked to PagerDuty, which alerts on-call engineers to the emergency. 
```

Comment:
I would probably update the above to be an actual list, e.g.:

Also you  have a typo on this line ("Incidident").

```
* hosting@humanmade.com is the most commonly used, and this linked to an Incident type (non-urgent).
* support@humanmade.com is rarely used, but this is linked to the Question support type.
* sysops@humanmade.com is the linked to an ‘Urgent’ Incident type, and is linked to PagerDuty, which alerts on-call engineers to the emergency. 
```

Reply:
Do we have Altis emails for this too? I think `humanmade.com` can be swapped for `altis-dxp.com`

Reply:
> Cleaned up some of the text and offered some suggestions for things. Mostly little stuff you can probably just click "commit suggestion" on 😁 but one comment at the end about the email addresses would need an actual update (because afaik I can't make multiline suggestions).

Thanks so much for the review! I'm not used to such turn-around times! :D 

> Do we have Altis emails for this too? I think humanmade.com can be swapped for altis-dxp.com

I didn't know we had those addresses, will update.